### PR TITLE
KAR-690 hindi & gujarati lang file updated

### PR DESCRIPTION
**Pull request by garima468**
KAR-690 hindi & Gujarati lang file updated

### CODE REVIEW

File: iGOT/client-assets/assets/i18n/hi.json [hunk 1]
```diff
@@ -20,26 +20,27 @@
     "knowledgeResources": "ज्ञान संसाधन",
     "dashboard": "डैशबोर्ड",
     "settings": "सेटिंग",
-    "lastAdded": "हाल में जोड़ा गया",
+    "lastAdded": "पिछला जोड़ा गया",
```
Comment (vishnubansaltarento) [inline iGOT/client-assets/assets/i18n/hi.json:23]: which one is correct?

File: iGOT/client-assets/assets/i18n/hi.json [hunk 2]
```diff
@@ -201,7 +221,7 @@
     "basedOnYourInterests": "आपकी रुचियों के आधार पर",
     "getReadyForLearningAdventure": "अपने अगले सीखने के रोमांच के लिए तैयार हो जाएँ।",
     "clickHere": "यहाँ क्लिक करें",
-    "toEnrollNextCourse": "अपने अगले पाठ्यक्रम में अभी नामांकन करने के लिए!",
+    "toEnrollNextCourse": "अब आपके अगले कोर्स में नामांकन करने के लिए",
```
Comment (vishnubansaltarento) [inline iGOT/client-assets/assets/i18n/hi.json:224]: which one is correct?

File: iGOT/client-assets/assets/i18n/hi.json [hunk 3]
```diff
@@ -335,7 +367,7 @@
   "apptocbanner": {
     "pleaseChooseBatch": "कोर्स शुरू करने के लिए कृपया एक बैच चुनें",
     "requestToEnroll": "नामांकन के लिए अनुरोध",
-    "lastEnrollmentDate": "नामांकन की अंतिम तिथि",
+    "lastEnrollmentDate": "पिछली नामांकन तिथि",
```
Comment (vishnubansaltarento) [inline iGOT/client-assets/assets/i18n/hi.json:370]: which one is correct?

File: iGOT/client-assets/assets/i18n/hi.json [hunk 4]
```diff
@@ -428,7 +460,7 @@
     "view": "देखें",
     "overallProgress": "समग्र प्रगति",
     "share": "शेयर करें।",
-    "items": "वस्तुएँ",
+    "items": "आइटम",
```
Comment (vishnubansaltarento) [inline iGOT/client-assets/assets/i18n/hi.json:463]: same as above?

File: iGOT/client-assets/assets/i18n/hi.json [hunk 5]
```diff
@@ -466,8 +498,9 @@
     "tryAgain": "फिर से कोशिश करें।",
     "mostTrending": "सबसे लोकप्रिय",
     "selfLearning": "स्वाध्याय",
-    "completedCourse": "आपने इस पाठ्यक्रम को पूरा किया",
-    "provider": "प्रोवाईडर"
+    "completedCourse": "आपने यह कोर्स पूरा लिया",
```
Comment (vishnubansaltarento) [inline iGOT/client-assets/assets/i18n/hi.json:501]: same as above?

File: iGOT/client-assets/assets/i18n/hi.json [hunk 6]
```diff
@@ -533,7 +566,7 @@
     "title3": "बलेंडेड कार्यक्रम खोलें",
     "description3": "उदाहरण के लिएः 'अनलाइन कोर्स बनाना और संचालित करना' खोजें।",
     "STEP4": "चरण 4",
-    "title4": "'उपस्थिति चिह्नित करने के लिए स्कैन' बटन दबाएँ।",
+    "title4": "'उपस्थिति दर्ज करने के लिए स्कैन' बटन दबाएँ।",
```
Comment (vishnubansaltarento) [inline iGOT/client-assets/assets/i18n/hi.json:569]: same as above?

File: iGOT/client-assets/assets/i18n/hi.json (hunk 1 above)
Comment (garima468) [reply iGOT/client-assets/assets/i18n/hi.json:23]: पिछला जोड़ा गया - this one is correct based on multilingual file

File: iGOT/client-assets/assets/i18n/hi.json (hunk 2 above)
Comment (garima468) [reply iGOT/client-assets/assets/i18n/hi.json:224]: अब आपके अगले कोर्स में नामांकन करने के लिए - this one is correct

File: iGOT/client-assets/assets/i18n/hi.json (hunk 3 above)
Comment (garima468) [reply iGOT/client-assets/assets/i18n/hi.json:370]: पिछली नामांकन तिथि - this one is correct

File: iGOT/client-assets/assets/i18n/hi.json (hunk 4 above)
Comment (garima468) [reply iGOT/client-assets/assets/i18n/hi.json:463]: आइटम - this one is correct

File: iGOT/client-assets/assets/i18n/hi.json (hunk 5 above)
Comment (garima468) [reply iGOT/client-assets/assets/i18n/hi.json:501]: आपने यह कोर्स पूरा लिया - this one is correct

File: iGOT/client-assets/assets/i18n/hi.json (hunk 6 above)
Comment (garima468) [reply iGOT/client-assets/assets/i18n/hi.json:569]: उपस्थिति दर्ज करने के लिए स्कैन' बटन दबाएँ। - this one is correct

**Pull request merged**